chain/network/src/types.rs

```diff
@@ -639,6 +639,8 @@ pub enum NetworkRequests {
 
     /// The following types of requests are used to trigger actions in the Peer Manager for testing.
     /// Fetch current routing table.
+    /// Used for testing purposes, unfortunately, this is now part of protocol.
+    #[cfg(feature = "test_features")]
```

Comment:
is it safe to 'remove' such entries from the enums? wouldn't it break the Borsch serialization of things below (like Sync or RequestUpdateNonce)